### ISSUE TITLE
Cacheability of 303 response (Trac #70)

### ISSUE THREAD

**Issue by mnot** [httpwg/httpbis-issues#70]
Section 10.3.4 says that 303 See Other is not cacheable.  

Migrated from https://trac.ietf.org/ticket/70
```json
{
    "status": "closed",
    "changetime": "2008-04-20T16:04:16",
    "_ts": "1208707456000000",
    "description": "Section 10.3.4 says that 303 See Other is not cacheable.  ",
    "reporter": "mnot@pobox.com",
    "cc": "",
    "resolution": "fixed",
    "time": "2007-12-22T02:57:13",
    "component": "p2-semantics",
    "summary": "Cacheability of 303 response",
    "priority": "",
    "keywords": "",
    "milestone": "03",
    "owner": "julian.reschke@gmx.de",
    "type": "design",
    "severity": ""
}
```

**Comment by mnot** [comment]
Trac update at 20071222T02:58:48: mnot@pobox.com commented:

> Proposal as per http://www.w3.org/mid/1FEA959C-00A9-4DE9-9597-B3338A6C915F@gbiv.com
> 
> 
>   10.3.4. 303 See Other
> 
>   The server directs the user agent to a different resource, indicated by a URI in the Location header field, that provides an indirect response to the original request. The user agent MAY perform a GET request on the URI in the Location field in order to obtain a representation corresponding to the response, be redirected again, or end with an error status. The Location URI is not a substitute reference for the originally requested resource.
> 
>   The 303 status is generally applicable to any HTTP method. It is primarily used to allow the output of a POST action to redirect the user agent to a selected resource, since doing so provides the information corresponding to the POST response in a form that can be separately identified, bookmarked, and cached independent of the original request.
> 
>   A 303 response to a GET request indicates that the requested resource does not have a representation of its own that can be transferred by the server over HTTP. The Location URI indicates a resource that is descriptive of the requested resource such that the follow-on representation may be useful without implying that that it adequately represents the previously requested resource. Note that answers to the questions of what can be represented, what representations are adequate, and what might be a useful description are outside the scope of HTTP and thus entirely determined by the resource owner(s).
> 
>   A 303 response SHOULD NOT be cached unless it is indicated as cacheable by Cache-Control or Expires header fields.  Except for responses to a HEAD request, the entity of a 303 response SHOULD contain a short hypertext note with a hyperlink to the Location URI.
>

**Comment by mnot** [comment]
Trac update at 20080104T06:05:18: 
* mnot@pobox.com changed component from "" to "cache"
* mnot@pobox.com changed milestone from "" to "unassigned"
* mnot@pobox.com changed owner from "" to ""

**Comment by mnot** [comment]
Trac update at 20080415T15:45:30: 
* julian.reschke@gmx.de changed component from "p6-cache" to "p2-semantics"
* julian.reschke@gmx.de changed owner from "" to "julian.reschke@gmx.de"

**Comment by mnot** [comment]
Trac update at 20080415T16:05:47: 
* julian.reschke@gmx.de commented:

> See [242].


* julian.reschke@gmx.de changed status from "new" to "assigned"

**Comment by mnot** [comment]
Trac update at 20080416T17:47:50: 
* mnot@pobox.com changed milestone from "unassigned" to "03"
* mnot@pobox.com changed resolution from "" to "fixed"
* mnot@pobox.com changed status from "assigned" to "closed"

**Comment by mnot** [comment]
Trac update at 20080420T16:04:16: julian.reschke@gmx.de commented:

> From [243]:
> 
> Fix typo (relates to #70).
>
